src/Neo.VM/Neo.VM.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.1;net7.0</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+</Project>
```

Comment:
Just a question: why the `<Copyright>`, `<Description>`, `<LangVersion>` and other description tags were removed? It is still a separate project, it would be nice to have some description metadata for it.

Reply:
This was the previous content. I don't think there is a lot to reuse. The older description was just "Neo virtual machine":
```
<PropertyGroup>
    <Copyright>2016-2023 The Neo Project</Copyright>
    <Description>Neo virtual machine</Description>
    <VersionPrefix>3.6.2</VersionPrefix>
    <Authors>The Neo Project</Authors>
    <TargetFrameworks>netstandard2.1;net7.0</TargetFrameworks>
    <LangVersion>9.0</LangVersion>
    <PackageTags>NEO;AntShares;Blockchain;Smart Contract;VM</PackageTags>
    <PackageProjectUrl>https://github.com/neo-project/neo-vm</PackageProjectUrl>
    <PackageLicenseExpression>MIT</PackageLicenseExpression>
    <RepositoryType>git</RepositoryType>
    <RepositoryUrl>https://github.com/neo-project/neo-vm.git</RepositoryUrl>
    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
    <Nullable>enable</Nullable>
    <GenerateDocumentationFile>true</GenerateDocumentationFile>
  </PropertyGroup>
```

Reply:
>  The older description was just

At least there was, so it would be nice to port it. As far as `Copyright`, `Authors` and package-related tags (we can change the `PackageProjectUrl` to `https://github.com/neo-project/neo`. VM is still a separate package, isn't it?

Reply:
Good, let's just leave it as is, resolved.